CHANGELOG.md

```diff
@@ -25,11 +25,20 @@ Here is a template for new release sections
 - `E-Land_Requirements.rst`: Official E-Land requirement list as well as progress on functional and non-functional requirements (#590)
 - Add pytests for `E4.detect_excessive_excess_generation_in_bus()` (#591)
 - Add pypi release to release protocol and update/simplify protocol (#601)
+- Remove REPORT_PATH constant (#607)
+- Add report assets and example simulation to package_data in `setup.py` (#607)
+- Add a util function to copy report asset to simulation output folder when user generates the report (#607)
+- Add entrypoints for `mvs_tool` and `mvs_report` in ´setup.py´ (#607)
 
 ### Changed
 - Order of readthedocs content (#590)
+- Make sure report can be generated even if figures are missing from simulation outputs (#607)
+- Move the code located in `mvs_report.py` into `multi_vector_simulator.cli:report` (#607)
+- Update installation steps in README and in RTD (#607)
+- If default folder does not exist when code is executed, example simulation's inputs are used from package_data (#607)
 
 ### Removed
+- Remove reference to git branch ID in the report (#607)
```

Comment:
Why?

Reply:
Because you are within this repository only when you execute the code from somewhere else there is no repository

Reply:
Ok, I got the idea. I hope there wont be too many mess-ups when we as devs simulate the pilots with the dev version. Maybe we should just prohibit people to do that xD